src/octoprint/cli/server.py

```diff
@@ -307,6 +309,7 @@ def get_value(key):
     safe_mode = "flag" if get_value("safe_mode") else None
     ignore_blacklist = get_value("ignore_blacklist")
     overlays = get_value("overlays")
+    no_color = bool(get_value("no_color"))
```

Comment:
Why are we changing the style of boolean? The safe mode uses None-not-None, and we have Python bool for no-color. Seems perhaps unnecessarily inconsistent, if I may bikeshed a little.

Reply:
`safe_mode` is a special case, as it doesn't set a boolean value but rather a source of the safe mode activation. If it is anything but `None`, OctoPrint will run in safe mode, but it will log the reason for that based on the value of this flag.

However, the explicit cast to `bool` should indeed not be needed here, as this should already be a boolean thanks to being marked with `is_flag=True` in the click def.